src/azul/plugins/metadata/hca/transform.py

```diff
@@ -274,6 +274,23 @@ def from_index(self, value: str) -> Optional[JSON]:
 value_and_unit: ValueAndUnit = ValueAndUnit()
 
 
+@attr.s(frozen=True, auto_attribs=True, kw_only=True)
+class Nested(PassThrough[JSON]):
+    es_type = 'nested'
+    properties: Mapping[str, FieldType]
+
+    # This allows for some level of interoperability between field types that
+    # are Nested vs Dictionaries.
+    def items(self):
+        return self.properties.items()
+
+
+accession: Nested = Nested(properties={
+    'namespace': null_str,
+    'accession': null_str
+})
```

Comment:
See if you can write a constructor with keyword arguments so that 

```suggestion
accession: Nested = Nested(namespace=null_str, accession=null_str)
```